src/app/pages/settings-integration/billing/settings-billing.component.ts

```diff
@@ -69,17 +67,16 @@ export class SettingsBillingComponent implements OnInit {
       }
     });
   }
-
-  public save(newSettings: any) {
+  public saveNewSettings(newSettings: any) {
     this.billingSettings.type = BillingSettingsType.STRIPE;
-    if (newSettings?.billing?.isTransactionBillingActivated) {
-      this.transactionBillingActivated = newSettings.billing.isTransactionBillingActivated;
-    } else {
-      this.transactionBillingActivated = this.billingSettings.billing.isTransactionBillingActivated;
-    }
-    this.billingSettings.billing = newSettings.billing as BillingSetting;
-    this.billingSettings.billing.isTransactionBillingActivated = this.transactionBillingActivated;
+    // Override current values with the new ones
```

Comment:
We should have the same approach (save, impl check)
![2021-06-23_16-36-02](https://user-images.githubusercontent.com/25266945/123116514-43c6d900-d441-11eb-9984-ece9560848d1.png)